test/dlc_tests/benchmark/sagemaker/tensorflow/training/test_trcomp_performance.py

```diff
@@ -175,3 +175,75 @@ def test_resnet101_at_fp16(self, instance_type, num_gpus, total_n_gpus, instance
             assert billable>=1000, 'False Positive '+result
             assert billable<=threshold, result
 
+@pytest.mark.flaky(reruns=1)
+@pytest.mark.usefixtures("sagemaker_only")
+class TestNLPModels:
+
+
+    @pytest.mark.model("facebook/bart-base")
```

Comment:
Why this choice of model ? We are looking for a model for which we can demonstrate good performance in a short period of time. I suggest GPT-2.